src/rdb/rdb_raft.c

```diff
@@ -2344,56 +2344,78 @@ rdb_raft_unload_lc(struct rdb *db)
 static int
 rdb_raft_get_election_timeout(void)
 {
-	const char     *s;
-	int		t;
+	char	       *name = "RDB_ELECTION_TIMEOUT";
+	unsigned int	default_value = 7000;
+	unsigned int	value = default_value;
 
-	s = getenv("RDB_ELECTION_TIMEOUT");
-	if (s == NULL)
-		t = 7000;
-	else
-		t = atoi(s);
-	return t;
+	d_getenv_int(name, &value);
+	if (value == 0 || value > INT_MAX) {
+		D_WARN("%s not in (0, %d] (defaulting to %u)\n", name, INT_MAX, default_value);
+		value = default_value;
+	}
+	return value;
 }
 
 static int
 rdb_raft_get_request_timeout(void)
 {
-	const char     *s;
-	int		t;
+	char	       *name = "RDB_REQUEST_TIMEOUT";
+	unsigned int	default_value = 3000;
+	unsigned int	value = default_value;
 
-	s = getenv("RDB_REQUEST_TIMEOUT");
-	if (s == NULL)
-		t = 3000;
-	else
-		t = atoi(s);
-	return t;
+	d_getenv_int(name, &value);
+	if (value == 0 || value > INT_MAX) {
+		D_WARN("%s not in (0, %d] (defaulting to %u)\n", name, INT_MAX, default_value);
+		value = default_value;
+	}
+	return value;
 }
 
 static uint64_t
-rdb_raft_get_compact_thres(void)
+rdb_raft_get_compact_thres()
```

Comment:
It doesn't seem like a big deal in general, but why are some functions using (void) in the signature and now others using only () ?

Reply:
Oops, it's an editing miss. Will fix...